pkg/config/application.go

```diff
@@ -0,0 +1,139 @@
+/*
+Copyright IBM Corp. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package config
+
+import (
+	"fmt"
+
+	cb "github.com/hyperledger/fabric-protos-go/common"
+	mb "github.com/hyperledger/fabric-protos-go/msp"
+	pb "github.com/hyperledger/fabric-protos-go/peer"
+)
+
+// Application encodes the application-level configuration needed in config
+// transactions
+type Application struct {
+	Organizations []*Organization
+	Capabilities  map[string]bool
+	Resources     *Resources
+	Policies      map[string]*Policy
+	ACLs          map[string]string
+}
+
+// AnchorPeer encodes the necessary fields to identify an anchor peer
+type AnchorPeer struct {
+	Host string
+	Port int
+}
+
+// NewApplicationGroup returns the application component of the channel configuration
+// It defines the organizations which are involved in application logic like chaincodes,
```

Comment:
nit: we should make sure to write our godoc comments in complete sentences.

Reply:
fixed 🔧